examples/react-app/SingleStream.jsx

```diff
@@ -1,6 +1,9 @@
 import React, { useState, useEffect } from 'react'
 
-import { Player } from 'media-stream-player'
+import { fetchTransformationMatrix, Player } from 'media-stream-player'
+import debug from 'debug'
+
+const debugLog = debug('msp:ws-sdp-video')
```

Comment:
I don't think `debug` is supposed to be used in the examples.

In the `fetchSDP` component, fine. But not here.

Reply:
Is it better to log it in console then?

Reply:
Yes, unless you can construct an actual example with it.

Reply:
An example has been written.

Reply:
Yes, you're not supposed to use this for examples, `msp:...` debugs should always come from within the library.

Just use a `console.log` in this case if it is to showcase, or use a different string alltogether.